core/src/main/java/org/apache/iceberg/MetadataUpdate.java

```diff
@@ -60,6 +66,11 @@ public int formatVersion() {
     public void applyTo(TableMetadata.Builder metadataBuilder) {
       metadataBuilder.upgradeFormatVersion(formatVersion);
     }
+
+    @Override
+    public void applyTo(ViewMetadata.Builder viewMetadataBuilder) {
+      viewMetadataBuilder.formatVersion(formatVersion);
```

Comment:
`upgradeFormatVersion`? That's what we use for tables because you can't downgrade. It's a little unclear what we would allow right now, but simply setting the format version is probably more permissive than we need.

Reply:
done